src/org/omegat/gui/exttrans/MachineTranslateTextArea.java

```diff
@@ -68,20 +78,30 @@
  * @author Jean-Christophe Helary
  * @author Aaron Madlon-Kay
  * @author Thomas Cordonnier
+ * @author Hiroshi Miura
  */
 @SuppressWarnings("serial")
 public class MachineTranslateTextArea extends EntryInfoThreadPane<MachineTranslationInfo>
         implements IPaneMenu {
 
     private static final String EXPLANATION = OStrings.getString("GUI_MACHINETRANSLATESWINDOW_explanation");
 
-    protected Set<MachineTranslationInfo> displayed = new HashSet<>();
+    /**
+     *  displayed hold entries.
+     *  <p>
+     *  It keeps the order by translatorName, MachineTranslationInfo is defined as Comparable&le;E&gt; that
+     *  prioritize translatorName. {@see MachineTranslationInfo}
+     *  A value should be unique and used for identify HTML element by id attribute.
+     *  We use an index counter for identifier of elements.
```

Comment:
It should be updated.
The PR does not sort an order of entries.

Reply:
Sort feature is implemented again at https://github.com/omegat-org/omegat/pull/205/commits/f606147e812e3a89e57ea174415a11f65d9f8398